Gemfile

```diff
@@ -3,4 +3,13 @@
 source "https://rubygems.org"
 
 # Specify your gem's dependencies in obs_github_deployments.gemspec
+gem "pry-byebug", "~> 3"
```

Comment:
Why are you moving the gems to the bundle?

Reply:
Because there is a cop encouraging us to do that. There is a lengthy discussion that explains why https://github.com/rubocop/rubocop/pull/11469#issuecomment-1401765614

Reply:
then remove the comment also please